python/neuroglancer/lib.py

```diff
@@ -272,6 +300,11 @@ def min2(a, b):
 def clamp(val, low, high):
   return min(max(val, low), high)
 
+def eclamp(val, low, high):
```

Comment:
It might be a better name. The use case for this function is where it's interchangeable with clamp. e.g. `fn = bounded ? eclamp : clamp`